assets/js/blocks/attribute-filter/block.js

```diff
@@ -170,7 +176,7 @@ const AttributeFilterBlock = ( { attributes } ) => {
 		<div className="wc-block-attribute-filter">
 			<CheckboxList
 				className={ 'wc-block-attribute-filter-list' }
-				options={ options }
+				options={ options ? options : [] }
```

Comment:
On problem with this construct is that you're always passing a NEW array along if `options` is a non truthy value.  So every call to this component by react to determine whether to re-render or not is going to trigger a re-render if nothing else changed (because we have a new array instance):

```js
render1Options = [];
render2Options = [];
render1Options !== render2Options
```

I'm not sure why you switched that anyways, as far as I can tell, options will always be an array?